h5io/_h5io.py

```diff
@@ -487,3 +499,31 @@ def list_file_contents(h5file):
         if not isinstance(h5file, h5py.File):
             raise TypeError(err.format(type(h5file)))
         _list_file_contents(h5file)
+
+
+def json_compatible(obj, slash='error'):
+    if isinstance(obj, (str, int, float, bool, type(None))):
```

Comment:
What about `unicode` on Python 2?

Reply:
I updated this to compare to `string_types` rather than `str`.